ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule/Shrinking.hs

```diff
@@ -78,19 +78,19 @@ trimBlockTree bt ps =
       where
         go youngest [] = youngest
         go youngest (block : blocks)
-          | any (`isYoungerThan` block) youngest = go youngest blocks
+          | any (`isDescendentOf` block) youngest = go youngest blocks
           | otherwise = go (block : youngest) blocks
 
     -- | Partial comparison of blocks. A block is older than another block if it
     -- is its ancestor. For test blocks, this can be seen in the hash.
-    isOlderThan :: TestBlock -> TestBlock -> Bool
-    isOlderThan b1 b2 =
+    isAncestorOf :: TestBlock -> TestBlock -> Bool
+    isAncestorOf b1 b2 =
```

Comment:
I suppose it would be helpful to put this function next to the TestBlock definition as it is used in other places. Could be done in another PR.

Reply:
Fair point; I'll handle it in this PR.

Reply:
Done in 018a85dc352b4dcfd31eddb878272e7a92faad86.